### DISCUSSION TITLE
问题

### DISCUSSION THREAD

**Discussion by mmmmmxx** [linguanyuan/server#2] [Q&A] [answered]
node通过哪些常用方法可以防止程序崩溃？

**Comment by linguanyuan** [comment, marked as answer]
通过以下方法可以防止程序崩溃。

（1） try-catch-finally。

（2） EventEmitter/Stream error事件处理。

（3） domain统一控制。

（4） jshint静态检查。

（5） jasmine/mocha单元测试
